### PR TITLE
Added 4 actions

### PR DESCRIPTION
**Pull request by koladiya**
create snapshot,
create tenant
get snapshots

### CODE REVIEW

**Comment by koladiya** [comment]
This three actions are added.

- create snapshot
- create tenant
- get snapshots
Checks were failing only in python27 environment.

**Comment by jinpingh** [comment]
Make sure  CHANGES.md and pack.yaml file are updated

**Comment by LindsayHill** [comment]
@koladiya check out the changes I made here - note the extra indentation required when breaking a line.

I highly recommend using an editor like VSCode, and installing the Python extension. Setup up piylint & flake8 locally, and your editor will show you the errors before you push, and it makes it a little easier to do the line breaks. Makes development turnaround faster when you can catch those before CI checks.

**Comment by koladiya** [comment]
@LindsayHill Thank you for fixing those indentations. I'll keep in mind this VSCode with extension next time.
I'll update the CHANGES.md soon.

File: pack.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ref: cisco_aci
 name: cisco_aci
 description: Cisco ACI pack
-version: 0.1.4
-author: Paul Mulvihill
-email: paul.mulvihill@pulsant.com
+version: 0.1.5
+author: Kamlesh Koladiya
+email: koladiya_111@yahoo.com
```
Comment (LindsayHill) [inline pack.yaml]: I don't think it's right to take over the original author here, unless they have given permission. Better to add a `contributors:` key, similar to this https://github.com/StackStorm-Exchange/stackstorm-vsphere/blob/master/pack.yaml#L9

**Review by LindsayHill — CHANGES_REQUESTED** [review]

File: pack.yaml (hunk 1 above)
Comment (koladiya) [reply pack.yaml]: Sorry about that. I updated the same.

**Review by LindsayHill — APPROVED** [review]

**Pull request merged**